packages/react/src/components/Alert/Alert.tsx

```diff
@@ -16,29 +16,61 @@
  * under the License.
  */
 
-import MuiAlert, {AlertProps as MuiAlertProps} from '@mui/material/Alert';
+import MuiAlert from '@mui/material/Alert';
+import type {AlertProps as MuiAlertProps} from '@mui/material/Alert';
+import type {OverridableComponent} from '@mui/material/OverridableComponent';
 import clsx from 'clsx';
-import {forwardRef, ForwardRefExoticComponent, ReactElement, MutableRefObject} from 'react';
+import {forwardRef} from 'react';
+import type {ReactElement, ElementType, Ref} from 'react';
 import type {WithWrapperProps} from '../../models/component';
 import composeComponentDisplayName from '../../utils/compose-component-display-name';
+import type {PaperTypeMap} from '../Paper';
 import './alert.scss';
 
-export type AlertProps = MuiAlertProps;
+export type AlertProps<C extends ElementType = ElementType> = {
+  /**
+   * The component used for the root node. Either a string to use a HTML element or a component.
+   */
+  component?: C;
+} & Omit<MuiAlertProps, 'component'>;
 
 const COMPONENT_NAME: string = 'Alert';
 
-const Alert: ForwardRefExoticComponent<AlertProps> & WithWrapperProps = forwardRef(
-  (props: AlertProps, ref: MutableRefObject<HTMLDivElement>): ReactElement => {
-    const {className, ...rest} = props;
-
+/**
+ * The Alert component display brief messages for the user without interrupting their use of the app.
+ *
+ * Demos:
+ *
+ * - [Alert (Oxygen UI)] (https://wso2.github.io/oxygen-ui/react/?path=/docs/feedback-alert)
```

Comment:
```suggestion
 * - [Alert (Oxygen UI)](https://wso2.github.io/oxygen-ui/react/?path=/docs/feedback-alert)
```

Reply:
The space is there in other places as well..

Reply:
There were several other places with this same problem.

Fixed all with https://github.com/wso2/oxygen-ui/pull/281/commits/262b1c99732e3a4da94f6b9f200aecaa8e09df74